test/system-test/system-tests/pkcs11_system_test.c

```diff
@@ -1949,7 +1949,7 @@ static CK_RV provisionPublicKey( CK_SESSION_HANDLE session,
         publicKeyTemplate[ 0 ].pValue = &class;
         publicKeyTemplate[ 1 ].pValue = &publicKeyType;
         publicKeyTemplate[ 2 ].pValue = &trueObject;
-        publicKeyTemplate[ 3 ].pValue = &modulus + 1;
+        publicKeyTemplate[ 3 ].pValue = &modulus[ 1 ];
```

Comment:
Can you update the Ci workflow file to enable ASAN in the system test run?

Reply:
This should be done in a separate PR.